### ISSUE TITLE
Increase readability top nav on campaign pages

### ISSUE THREAD

**Issue by crystalruthbell** [DoSomethingArchive/legacy-website#1776]
@weerd I really like the subtle shadow you added to campaign text and taglines and think this sort of a solution could be a good fit for campaign pages. I think that adding a toggle for white / black nav that content creators can manipulate might also be a good solution. 

Lets discuss in our design/frontend brainstorm!

![screen shot 2014-04-14 at 1 35 43 pm](https://cloud.githubusercontent.com/assets/6550604/2698524/96558904-c3fb-11e3-8d4b-e73992849f3f.png)
![screen shot 2014-04-14 at 1 38 09 pm](https://cloud.githubusercontent.com/assets/6550604/2698525/9655ec3c-c3fb-11e3-9611-9c0fc096930f.png)
![screen shot 2014-04-14 at 1 37 47 pm](https://cloud.githubusercontent.com/assets/6550604/2698527/96563944-c3fb-11e3-9bbc-5c7bb7fcaecf.png)
![screen shot 2014-04-14 at 1 37 32 pm](https://cloud.githubusercontent.com/assets/6550604/2698526/96563174-c3fb-11e3-8cea-ffb63c448db7.png)

**Comment by barryclark** [comment]
We're going to go with...
- [ ] A toggle on the image not for light or dark nav
- [x] A shadow on the text in the navigation
- [ ] Look into image processing to see if we can do this automagically, with something like ImageMagick. cc @angaither

**Comment by weerd** [comment]
I've provided a temporary solution until I can undertake the Nav refactoring (hopefully soon). This should help with legibility for the time being. I added the same black dropshadow when the nav text is white, and a slightly modified white dropshadow when the text in the nav is black.

See examples below:

![screen shot 2014-05-29 at 11 07 07 am](https://cloud.githubusercontent.com/assets/105849/3119298/08461bc0-e745-11e3-81d8-68785e3fe6ba.png)

![screen shot 2014-05-29 at 11 07 29 am](https://cloud.githubusercontent.com/assets/105849/3119300/0c457f36-e745-11e3-8ff9-85e2e0531c03.png)
